QUARKUS-2479.md

```diff
@@ -0,0 +1,28 @@
+# QUARKUS-2479 SmallRye Config SecretKeys support
+
+Jira: https://issues.redhat.com/browse/QUARKUS-SmallRye Config SecretKeys support
+
+This test plan covers support of secret keys in SmallRye config
+
+## Scope of the testing
+Following topics should be covered:
+
+1. Verify, that some secrets can be marked as secret 
+2. Verify, that usage of these secrets leads to an exception
+3. Verify, that these secrets can be used only within doUnlocked section
+
+### Impact on test suites and testing automation
+- One new test class added for Kafka Dev Services
+- One test method in sql/hibernate reactive refactored 
+
+### Impact on resources:
+
+- No additional requirements for resources in lab
+- Required additional time for the test execution should be less, than minute.
```

Comment:
I think that we should cover also native mode at least in `sql/hibernate reactive` just to verify that the new classes that we are using on this feature support mandrel / graalVM compilation. 

I am not sure if OpenShift coverage adds any value to this feature, WDYT ?

Reply:
I was talking with @rsvoboda about OCP coverage and we believe that from now baremetal is enough.

Reply:
Added considerations about native compilation